index.html

```diff
@@ -9141,13 +9139,21 @@ <h3>Dispatching actions</h3>
    session types can allow running multiple commands in parallel, in
    which case this is necessary to ensure sequential access.
  </aside>
+</ol>
 
- <li><p>Let <var>actions result</var> be the result of <a>dispatch
- actions inner</a> with <var>input state</var>, <var>actions by
- tick</var>, <var>browsing context</var>, and <var>actions options</var>.
+<p>To <dfn>dispatch actions</dfn> given <var>input
+state</var>, <var>actions by tick</var>, <var>browsing
+context</var>, and <var>actions options</var>:
+
+<ol class=algorithm>
+ <li><p><a>Wait for an action queue token</a> with <var>input state</var>.
+
+ <li><p>Let <var>actions result</var> be the result of
+  <a>dispatch actions inner</a> with <var>input state</var>, <var>actions by
+  tick</var>, <var>browsing context</var>, and <var>actions options</var>.
 
  <li><p>Dequeue <var>input state</var>&apos;s <a>actions queue</a>.
-     <p>Assert: this returns <var>token</var>
+  <p>Assert: this returns <var>token</var>
```

Comment:
Now that "wait for an action queue token" is a separate algorithm, there is no `token` variable here in "dispatch actions".